rootfs/etc/nginx/lua/util/nodemap.lua

```diff
@@ -0,0 +1,120 @@
+local math = require("math")
+local util = require("util")
```

Comment:
`tablelength` seems to be the only function you use from utils, for better locality (hence perf) you can do

```
local tablelength = require("util").tablelength
```

and use `tablelength` directly